website/components/EditLink.tsx

```diff
@@ -0,0 +1,38 @@
+import react from "react";
+
+const excludedPaths = ["pages/plugins/index.mdx"];
+
+export function EditLink({
+  className,
+  filePath,
+  children,
+}: {
+  className?: string;
+  filePath?: string;
+  children: react.ReactNode;
+}) {
+  if (!filePath) {
```

Comment:
The ability to control the edit on GitHub link is the main reason to do this update